locales/en/metrics.json

```diff
@@ -34,6 +34,8 @@
   "metric_kpi_topics_description": "Topics are event logs in a $t(common:kafka) instance. This metric shows the total number of topics in the $t(common:kafka) instance.",
   "metric_kpi_topics_name": "Topics",
   "metric_not_available": "Data unavailable",
+  "metrics_lag_description": "Metrics regularly experience lag, and do not automatically refresh. This might result in metrics appearing out-of-sync and with details displayed on other pages.",
```

Comment:
One more typo here that I missed 😵‍💫 
The "and" should be removed from the second statement:
> Metrics regularly experience lag, and do not automatically refresh. This might result in metrics appearing out-of-sync ~and~ with details displayed on other pages.